libs/sdk-ui-charts/src/charts/headline/internal/providers/ComparisonProvider.ts

```diff
@@ -1,11 +1,79 @@
 // (C) 2023 GoodData Corporation
 import React from "react";
+
+import {
+    ArithmeticMeasureOperator,
+    bucketMeasure,
+    bucketsFind,
+    IBucket,
+    IMeasure,
+    isPoPMeasure,
+    isPreviousPeriodMeasure,
+    newArithmeticMeasure,
+    newBucket,
+} from "@gooddata/sdk-model";
+
 import { IHeadlineTransformationProps } from "../../HeadlineProvider.js";
 import AbstractProvider from "./AbstractProvider.js";
+import ComparisonTransformation from "../transformations/ComparisonTransformation.js";
+import { CalculationType, IComparison } from "../../../../interfaces/index.js";
+import { BucketNames } from "@gooddata/sdk-ui";
+
+const ARITHMETIC_BUCKET_IDENTIFIER = "comparison_arithmetic_bucket_identifier";
```

Comment:
I would name the value just `"comparison_arithmetic_bucket"`.

Reply:
Changed!